### PR TITLE
feature/Add files to ignore

### PR DESCRIPTION
**Pull request by parisbs**
* Add new files to be ignored by git

### CODE REVIEW

**Comment by Jesus95I** [comment]
LGTM!

**Review by Jesus95I — APPROVED** [review]

**Comment by alexisgmz12** [comment]
Nop, llora

File: .gitignore [hunk 1]
```diff
@@ -52,8 +52,8 @@ captures/
 
 # Keystore files
 # Uncomment the following lines if you do not want to check your keystore files in.
-#*.jks
-#*.keystore
+*.jks
```
Comment (parisbs) [inline .gitignore:55]: Required to avoid security issues

File: .gitignore [hunk 2]
```diff
@@ -83,3 +83,7 @@ lint/generated/
 lint/outputs/
 lint/tmp/
 # lint/reports/
+
+*.txt
+miruta/
+ejemplo.jpg
```
Comment (parisbs) [inline .gitignore:89]: Unnecessary file

**Pull request merged**